# 02j – 11. Experiment: Post-Bugfix: Hyperparameter-Grid BL4 (beam_size × max_length)

## Kontext: Der Bugfix

In `01a_` wurde ein Bug in `segmentation.py` entdeckt und behoben:

```python
# VORHER (fehlerhaft): min_duration_off las den Wert von min_duration_ON
min_duration_off_frames = int(parameters.get("min_duration_off",
    CENTRAL_ASD_CHUNKING_PARAMETERS["min_duration_on"]) * 25)  # ← falscher Key!

# NACHHER (korrekt):
min_duration_off_frames = int(parameters.get("min_duration_off",
    CENTRAL_ASD_CHUNKING_PARAMETERS["min_duration_off"]) * 25)  # ← richtiger Key
```

Der Bug bedeutete, dass der Fallback-Wert für `min_duration_off` (Mindeststille
zwischen zwei Sprachsegmenten) falsch war. Nach dem Bugfix kann
die Segmentierung kürze Pausen korrekt handhaben.

## Ziel dieses Notebooks

Das Hyperparameter-Grid aus `02_` (beam_size × max_length) wird nach dem Bugfix
neu durchlaufen, um zu prüfen ob die optimalen Parameterkombinationen
nach der Bugbehebung die gleichen sind.

## Ergebnis (Vorschau)

| beam_size | max_length | WER | Joint Error |
|-----------|-----------|-----|-------------|
| 4 | 10 s | ~0.575 | – |
| 8 | 15 s | ~0.548 | – |
| 12 | 20 s | **0.524** | **0.338** |

Beste Kombination nach Bugfix: **beam=12, len=20** – gleiche Tendenz wie vor dem Bugfix,
aber die absoluten WER-Werte haben sich leicht verändert.

**Hinweis Bugfix-Kontext:** Alle vorherigen Experimente (`02_`–`02i_`) liefen **ohne** diesen Bugfix und zeigten, dass `beam=12, len=20` die beste Decoding-Konfiguration ist. Dieses Notebook reevaluiert diese Konfiguration **nach dem Bugfix** – und stellt fest, dass der Bugfix allein die WER leicht verschlechtert. Deshalb folgt in `02k_` der `min_duration`-Grid-Search, der den Bugfix-Nachteil mehr als ausgleicht.

## 1 – GPU-Check & Auswahl

In [2]:
!nvidia-smi

Fri Jan 30 16:28:45 2026       
+-----------------------------------------------------------------------------------------+
| NVIDIA-SMI 550.54.15              Driver Version: 550.54.15      CUDA Version: 12.5     |
|-----------------------------------------+------------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id          Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |           Memory-Usage | GPU-Util  Compute M. |
|                                         |                        |               MIG M. |
|=========================================+========================+======================|
|   0  NVIDIA A100-SXM4-80GB          On  |   00000000:01:00.0 Off |                    0 |
| N/A   34C    P0             80W /  500W |    7093MiB /  81920MiB |      0%      Default |
|                                         |                        |             Disabled |
+-----------------------------------------+-----

In [3]:
import os

# Physische GPU-Auswahl: hier GPU 2 (siehe nvidia-smi)
os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
os.environ["CUDA_VISIBLE_DEVICES"] = "2"


## 2 – CUDA-Verifikation

In [4]:
import torch

In [5]:
print("CUDA devices:", torch.cuda.device_count())
print("Device 0 name (sollte A100 sein):", torch.cuda.get_device_name(0))

CUDA devices: 1
Device 0 name (sollte A100 sein): NVIDIA A100-SXM4-80GB


## 3 – Setup: Arbeitsverzeichnis & Imports

In [6]:
import os, sys
import pandas as pd

# Arbeitsverzeichnis auf Repo-Root setzen (Voraussetzung für alle relativen Pfade)
project_baseline_path = "/home/josch080/Projektgruppe/mcorec_baseline"
os.chdir(project_baseline_path)

# Repo-Root in sys.path, damit projektinterne Module importierbar sind
if project_baseline_path not in sys.path:
    sys.path.append(project_baseline_path)

from script.pg_utils_experiments import run_inference_for_experiment, run_eval_and_log, append_eval_results_for_experiments

/home/josch080/Projektgruppe/mcorec_baseline/script/inference.py:22: FutureWarning: In the future `np.object` will be defined as the corresponding NumPy scalar.
  if not hasattr(np, "object"):


## 4 – Modell-Definitionen

Alle Baseline-Modelle sind hier für Vollständigkeit definiert,
aber nur `avsr_cocktail_finetuned` (BL4) wird in den Experimenten genutzt.

In [7]:
BASE_MODELS = {
    "auto_avsr": {
        "model_type": "auto_avsr",
        "chkpt": "model-bin/auto_avsr/avsr_trlrwlr2lrs3vox2avsp_base.pth",
    },
    "avsr_cocktail": {
        "model_type": "avsr_cocktail",
        "chkpt": "model-bin/avsr_cocktail",
    },
    "avsr_cocktail_finetuned": {
        "model_type": "avsr_cocktail",
        "chkpt": "model-bin/avsr_cocktail_mcorec_finetune",
    },
    "muavic_en": {
        "model_type": "muavic_en",
        "chkpt": "nguyenvulebinh/AV-HuBERT-MuAViC-en",
    },
}


## 5 – Sessions & Experimente

Gleiches 5-Session-Dev-Subset wie in allen vorherigen Notebooks.
Das Grid (E44–E54) repliziert das Grid aus `02_`, nun mit korrekt gefixter Segmentierung.

E54 entspricht den ursprünglichen BL4-Baseline-Parametern (beam=3, len=15) –
dient als direkter Vergleichspunkt vor/nach Bugfix.

In [8]:
SESSION_IDS = ["session_40", "session_43", "session_49", "session_50", "session_54"]

In [9]:
EXPERIMENTS = {
    # 15 Sekunden
    "E44_bugfix_bs4_len15": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 4,
        "max_length": 15,
        "comment": "beam=4, len=15",
    },
    "E45_bugfix_bs8_len15": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 8,
        "max_length": 15,
        "comment": "beam=8, len=15",
    },
    "E46_bugfix_bs12_len15": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 12,
        "max_length": 15,
        "comment": "beam=12, len=15",
    },

    # 18 Sekunden
    "E47_bugfix_bs4_len18": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 4,
        "max_length": 18,
        "comment": "beam=4, len=18",
    },
    "E48_bugfix_bs8_len18": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 8,
        "max_length": 18,
        "comment": "beam=8, len=18",
    },
    "E49_bugfix_bs12_len18": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 12,
        "max_length": 18,
        "comment": "beam=12, len=18",
    },

    # 20 Sekunden
    "E50_bugfix_bs4_len20": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 4,
        "max_length": 20,
        "comment": "beam=4, len=20",
    },
    "E51_bugfix_bs8_len20": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 8,
        "max_length": 20,
        "comment": "beam=8, len=20",
    },
    "E52_bugfix_bs12_len20": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 12,
        "max_length": 20,
        "comment": "beam=12, len=20",
    },

    "E53_bugfix_bs8_len10": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 8,
        "max_length": 10,
        "comment": "beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)",
    },
    "E54_bugfix_bs3_len15": {
        "base_model": "avsr_cocktail_finetuned",
        "beam_size": 3,
        "max_length": 15,
        "comment": "beam=3, len=15 (Baseline mit Bugfix)",
    },
}


## 6 – Inference

11 Experimente × 5 Sessions = 55 Läufe.

In [10]:
for sid in SESSION_IDS:
    session_dir = f"data-bin/dev/{sid}"
    print(f"\n########## Starte Experimente für {sid} ##########")

    for exp_name in EXPERIMENTS:
        run_inference_for_experiment(
            exp_name=exp_name,
            base_models=BASE_MODELS,
            experiments=EXPERIMENTS,
            session_dir=session_dir,
        )


########## Starte Experimente für session_40 ##########

Starte Inference für Experiment: E44_bugfix_bs4_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 15
  output_dir_name = output_E44_bugfix_bs4_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=4, len=15
Loading avsr_cocktail model...


/home/josch080/Projektgruppe/mcorec_train/lib/python3.11/site-packages/tqdm/auto.py:21: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html
  from .autonotebook import tqdm as notebook_tqdm


Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:35<02:56, 35.31s/it]


Processing speakers:  33%|███▎      | 2/6 [01:51<03:56, 59.12s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 17/17 [00:32<00:00,  1.92s/it]

Processing speakers:  50%|█████     | 3/6 [02:57<03:07, 62.62s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:35<00:00,  1.77s/it]

Processing speakers:  67%|██████▋   | 4/6 [03:55<02:00, 60.45s/it]


Processing speakers:  83%|████████▎ | 5/6 [04:43<00:56, 56.11s/it]


Processing speakers: 100%|██████████| 6/6 [05:42<00:00, 57.04s/it]



Starte Inference für Experiment: E45_bugfix_bs8_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 15
  output_dir_name = output_E45_bugfix_bs8_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=8, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:38<03:14, 38.97s/it]


Processing speakers:  33%|███▎      | 2/6 [01:54<04:02, 60.55s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 17/17 [00:36<00:00,  2.13s/it]

Processing speakers:  50%|█████     | 3/6 [03:06<03:17, 65.90s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:37<00:00,  1.87s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:06<02:06, 63.48s/it]


Processing speakers:  83%|████████▎ | 5/6 [04:55<00:58, 58.38s/it]


Processing speakers: 100%|██████████| 6/6 [05:57<00:00, 59.51s/it]



Starte Inference für Experiment: E46_bugfix_bs12_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 15
  output_dir_name = output_E46_bugfix_bs12_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=12, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:41<03:27, 41.49s/it]


Processing speakers:  33%|███▎      | 2/6 [02:02<04:18, 64.55s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 17/17 [00:39<00:00,  2.30s/it]

Processing speakers:  50%|█████     | 3/6 [03:20<03:32, 70.70s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:39<00:00,  1.98s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:24<02:16, 68.14s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:19<01:03, 63.44s/it]


Processing speakers: 100%|██████████| 6/6 [06:22<00:00, 63.78s/it]



Starte Inference für Experiment: E47_bugfix_bs4_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 18
  output_dir_name = output_E47_bugfix_bs4_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=4, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:33<02:48, 33.61s/it]


Processing speakers:  33%|███▎      | 2/6 [01:44<03:42, 55.73s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:36<00:00,  2.30s/it]

Processing speakers:  50%|█████     | 3/6 [02:56<03:09, 63.10s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:35<00:00,  1.79s/it]

Processing speakers:  67%|██████▋   | 4/6 [03:55<02:02, 61.24s/it]


Processing speakers:  83%|████████▎ | 5/6 [04:50<00:59, 59.01s/it]


Processing speakers: 100%|██████████| 6/6 [05:51<00:00, 58.61s/it]



Starte Inference für Experiment: E48_bugfix_bs8_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 18
  output_dir_name = output_E48_bugfix_bs8_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=8, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:37<03:07, 37.44s/it]


Processing speakers:  33%|███▎      | 2/6 [01:56<04:06, 61.74s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:40<00:00,  2.52s/it]

Processing speakers:  50%|█████     | 3/6 [03:12<03:25, 68.37s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:38<00:00,  1.94s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:11<02:09, 64.84s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:08<01:01, 61.77s/it]


Processing speakers: 100%|██████████| 6/6 [06:13<00:00, 62.24s/it]



Starte Inference für Experiment: E49_bugfix_bs12_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 18
  output_dir_name = output_E49_bugfix_bs12_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=12, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:41<03:27, 41.55s/it]


Processing speakers:  33%|███▎      | 2/6 [02:04<04:23, 65.78s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:42<00:00,  2.66s/it]

Processing speakers:  50%|█████     | 3/6 [03:23<03:36, 72.06s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:42<00:00,  2.10s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:29<02:18, 69.44s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:35<01:08, 68.18s/it]


Processing speakers: 100%|██████████| 6/6 [06:45<00:00, 67.56s/it]



Starte Inference für Experiment: E50_bugfix_bs4_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 20
  output_dir_name = output_E50_bugfix_bs4_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=4, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:33<02:49, 34.00s/it]


Processing speakers:  33%|███▎      | 2/6 [01:50<03:57, 59.28s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:36<00:00,  2.30s/it]

Processing speakers:  50%|█████     | 3/6 [03:06<03:19, 66.47s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 19/19 [00:37<00:00,  1.97s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:05<02:07, 63.79s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:02<01:01, 61.43s/it]


Processing speakers: 100%|██████████| 6/6 [06:02<00:00, 60.34s/it]



Starte Inference für Experiment: E51_bugfix_bs8_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 20
  output_dir_name = output_E51_bugfix_bs8_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=8, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:36<03:04, 36.94s/it]


Processing speakers:  33%|███▎      | 2/6 [02:00<04:17, 64.37s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:41<00:00,  2.58s/it]

Processing speakers:  50%|█████     | 3/6 [03:23<03:39, 73.06s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 19/19 [00:42<00:00,  2.22s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:27<02:18, 69.20s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:26<01:05, 65.58s/it]


Processing speakers: 100%|██████████| 6/6 [06:31<00:00, 65.17s/it]



Starte Inference für Experiment: E52_bugfix_bs12_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 20
  output_dir_name = output_E52_bugfix_bs12_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=12, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:41<03:25, 41.05s/it]


Processing speakers:  33%|███▎      | 2/6 [02:06<04:29, 67.46s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 16/16 [00:43<00:00,  2.69s/it]

Processing speakers:  50%|█████     | 3/6 [03:34<03:49, 76.45s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 19/19 [00:45<00:00,  2.37s/it]

Processing speakers:  67%|██████▋   | 4/6 [04:42<02:26, 73.07s/it]


Processing speakers:  83%|████████▎ | 5/6 [05:46<01:09, 69.80s/it]


Processing speakers: 100%|██████████| 6/6 [06:53<00:00, 68.89s/it]



Starte Inference für Experiment: E53_bugfix_bs8_len10
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 10
  output_dir_name = output_E53_bugfix_bs8_len10
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:36<03:04, 36.80s/it]


Processing speakers:  33%|███▎      | 2/6 [01:42<03:35, 53.81s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 20/20 [00:29<00:00,  1.49s/it]

Processing speakers:  50%|█████     | 3/6 [02:44<02:53, 57.67s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 24/24 [00:37<00:00,  1.55s/it]

Processing speakers:  67%|██████▋   | 4/6 [03:39<01:53, 56.57s/it]


Processing speakers:  83%|████████▎ | 5/6 [04:31<00:54, 54.76s/it]


Processing speakers: 100%|██████████| 6/6 [05:31<00:00, 55.29s/it]



Starte Inference für Experiment: E54_bugfix_bs3_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 3
  max_length      = 15
  output_dir_name = output_E54_bugfix_bs3_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_40
  comment         = beam=3, len=15 (Baseline mit Bugfix)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_40


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:32<02:40, 32.00s/it]


Processing speakers:  33%|███▎      | 2/6 [01:40<03:34, 53.55s/it]


Processing speaker spk_2 track 1 of 3: 0it [00:00, ?it/s]

Processing speaker spk_2 track 2 of 3: 100%|██████████| 17/17 [00:34<00:00,  2.01s/it]

Processing speakers:  50%|█████     | 3/6 [02:47<02:58, 59.46s/it]


Processing speaker spk_3 track 1 of 2: 100%|██████████| 20/20 [00:34<00:00,  1.74s/it]

Processing speakers:  67%|██████▋   | 4/6 [03:40<01:53, 56.96s/it]


Processing speakers:  83%|████████▎ | 5/6 [04:26<00:53, 53.28s/it]


Processing speakers: 100%|██████████| 6/6 [05:24<00:00, 54.03s/it]



########## Starte Experimente für session_43 ##########

Starte Inference für Experiment: E44_bugfix_bs4_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 15
  output_dir_name = output_E44_bugfix_bs4_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=4, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 32/32 [00:51<00:00,  1.61s/it]

Processing speakers:  17%|█▋        | 1/6 [01:06<05:31, 66.38s/it]


Processing speakers:  33%|███▎      | 2/6 [01:43<03:17, 49.33s/it]


Processing speakers:  50%|█████     | 3/6 [02:31<02:26, 48.67s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:31<01:45, 52.89s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:19<00:00,  1.15s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:12<00:48, 48.79s/it]


Processing speakers: 100%|██████████| 6/6 [04:50<00:00, 48.41s/it]



Starte Inference für Experiment: E45_bugfix_bs8_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 15
  output_dir_name = output_E45_bugfix_bs8_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=8, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 32/32 [00:54<00:00,  1.70s/it]

Processing speakers:  17%|█▋        | 1/6 [01:10<05:51, 70.36s/it]


Processing speakers:  33%|███▎      | 2/6 [01:51<03:32, 53.16s/it]


Processing speakers:  50%|█████     | 3/6 [02:45<02:40, 53.45s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:50<01:56, 58.11s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:20<00:00,  1.23s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:36<00:53, 53.69s/it]


Processing speakers: 100%|██████████| 6/6 [05:16<00:00, 52.73s/it]



Starte Inference für Experiment: E46_bugfix_bs12_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 15
  output_dir_name = output_E46_bugfix_bs12_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=12, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 32/32 [00:55<00:00,  1.75s/it]

Processing speakers:  17%|█▋        | 1/6 [01:12<06:01, 72.35s/it]


Processing speakers:  33%|███▎      | 2/6 [01:56<03:42, 55.58s/it]


Processing speakers:  50%|█████     | 3/6 [02:55<02:51, 57.06s/it]


Processing speakers:  67%|██████▋   | 4/6 [04:02<02:02, 61.30s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:23<00:00,  1.38s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:50<00:56, 56.57s/it]


Processing speakers: 100%|██████████| 6/6 [05:35<00:00, 55.87s/it]



Starte Inference für Experiment: E47_bugfix_bs4_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 18
  output_dir_name = output_E47_bugfix_bs4_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=4, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [00:54<00:00,  1.83s/it]

Processing speakers:  17%|█▋        | 1/6 [01:14<06:14, 74.92s/it]


Processing speakers:  33%|███▎      | 2/6 [01:52<03:31, 52.97s/it]


Processing speakers:  50%|█████     | 3/6 [02:42<02:34, 51.45s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:44<01:51, 55.92s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:18<00:00,  1.11s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:24<00:50, 50.16s/it]


Processing speakers: 100%|██████████| 6/6 [05:01<00:00, 50.28s/it]



Starte Inference für Experiment: E48_bugfix_bs8_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 18
  output_dir_name = output_E48_bugfix_bs8_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=8, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [00:58<00:00,  1.94s/it]

Processing speakers:  17%|█▋        | 1/6 [01:19<06:36, 79.37s/it]


Processing speakers:  33%|███▎      | 2/6 [01:59<03:45, 56.40s/it]


Processing speakers:  50%|█████     | 3/6 [02:52<02:44, 54.83s/it]


Processing speakers:  67%|██████▋   | 4/6 [04:00<01:59, 59.94s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:21<00:00,  1.26s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:46<00:54, 54.80s/it]


Processing speakers: 100%|██████████| 6/6 [05:25<00:00, 54.30s/it]



Starte Inference für Experiment: E49_bugfix_bs12_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 18
  output_dir_name = output_E49_bugfix_bs12_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=12, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [01:02<00:00,  2.07s/it]

Processing speakers:  17%|█▋        | 1/6 [01:23<06:56, 83.29s/it]


Processing speakers:  33%|███▎      | 2/6 [02:05<03:56, 59.13s/it]


Processing speakers:  50%|█████     | 3/6 [03:02<02:54, 58.07s/it]


Processing speakers:  67%|██████▋   | 4/6 [04:18<02:10, 65.14s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:22<00:00,  1.32s/it]

Processing speakers:  83%|████████▎ | 5/6 [05:05<00:58, 58.64s/it]


Processing speakers: 100%|██████████| 6/6 [05:49<00:00, 58.22s/it]



Starte Inference für Experiment: E50_bugfix_bs4_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 20
  output_dir_name = output_E50_bugfix_bs4_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=4, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [00:53<00:00,  1.78s/it]

Processing speakers:  17%|█▋        | 1/6 [01:13<06:08, 73.74s/it]


Processing speakers:  33%|███▎      | 2/6 [01:51<03:29, 52.43s/it]


Processing speakers:  50%|█████     | 3/6 [02:42<02:35, 51.71s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:43<01:51, 55.68s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:18<00:00,  1.10s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:25<00:50, 50.53s/it]


Processing speakers: 100%|██████████| 6/6 [05:00<00:00, 50.09s/it]



Starte Inference für Experiment: E51_bugfix_bs8_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 20
  output_dir_name = output_E51_bugfix_bs8_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=8, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [00:55<00:00,  1.86s/it]

Processing speakers:  17%|█▋        | 1/6 [01:17<06:25, 77.16s/it]


Processing speakers:  33%|███▎      | 2/6 [01:56<03:40, 55.11s/it]


Processing speakers:  50%|█████     | 3/6 [02:53<02:46, 55.63s/it]


Processing speakers:  67%|██████▋   | 4/6 [04:00<02:00, 60.23s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:22<00:00,  1.34s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:46<00:55, 55.14s/it]


Processing speakers: 100%|██████████| 6/6 [05:27<00:00, 54.63s/it]



Starte Inference für Experiment: E52_bugfix_bs12_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 20
  output_dir_name = output_E52_bugfix_bs12_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=12, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 30/30 [01:01<00:00,  2.04s/it]

Processing speakers:  17%|█▋        | 1/6 [01:20<06:41, 80.29s/it]


Processing speakers:  33%|███▎      | 2/6 [02:04<03:55, 58.90s/it]


Processing speakers:  50%|█████     | 3/6 [03:04<02:59, 59.75s/it]


Processing speakers:  67%|██████▋   | 4/6 [04:18<02:09, 65.00s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:22<00:00,  1.32s/it]

Processing speakers:  83%|████████▎ | 5/6 [05:06<00:58, 58.97s/it]


Processing speakers: 100%|██████████| 6/6 [05:50<00:00, 58.47s/it]



Starte Inference für Experiment: E53_bugfix_bs8_len10
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 10
  output_dir_name = output_E53_bugfix_bs8_len10
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 33/33 [00:51<00:00,  1.57s/it]

Processing speakers:  17%|█▋        | 1/6 [01:06<05:32, 66.48s/it]


Processing speakers:  33%|███▎      | 2/6 [01:44<03:19, 49.87s/it]


Processing speakers:  50%|█████     | 3/6 [02:33<02:27, 49.27s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:31<01:45, 52.76s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 18/18 [00:19<00:00,  1.09s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:14<00:49, 49.15s/it]


Processing speakers: 100%|██████████| 6/6 [04:53<00:00, 48.87s/it]



Starte Inference für Experiment: E54_bugfix_bs3_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 3
  max_length      = 15
  output_dir_name = output_E54_bugfix_bs3_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_43
  comment         = beam=3, len=15 (Baseline mit Bugfix)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_43


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 32/32 [00:49<00:00,  1.53s/it]

Processing speakers:  17%|█▋        | 1/6 [01:04<05:23, 64.60s/it]


Processing speakers:  33%|███▎      | 2/6 [01:39<03:08, 47.08s/it]


Processing speakers:  50%|█████     | 3/6 [02:26<02:21, 47.09s/it]


Processing speakers:  67%|██████▋   | 4/6 [03:24<01:42, 51.39s/it]


Processing speaker spk_4 track 1 of 2: 100%|██████████| 17/17 [00:18<00:00,  1.09s/it]

Processing speakers:  83%|████████▎ | 5/6 [04:05<00:47, 47.52s/it]


Processing speakers: 100%|██████████| 6/6 [04:39<00:00, 46.51s/it]



########## Starte Experimente für session_49 ##########

Starte Inference für Experiment: E44_bugfix_bs4_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 15
  output_dir_name = output_E44_bugfix_bs4_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=4, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:12<01:04, 12.91s/it]


Processing speakers:  33%|███▎      | 2/6 [00:34<01:12, 18.25s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.22it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.00it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.31it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 4/4 [00:12<00:00,  3.08s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 3/3 [00:06<00:00,  2.26s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:15<00:00,  1.71s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.14it/s]

Processing speakers:  50%|█████     | 3/6 [01:27<01:41, 33.76s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:05<01:11, 35.68s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:55<00:40, 40.80s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 26/26 [00:32<00:00,  1.24s/it]

Processing speakers: 100%|██████████| 6/6 [03:41<00:00, 36.88s/it]



Starte Inference für Experiment: E45_bugfix_bs8_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 15
  output_dir_name = output_E45_bugfix_bs8_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=8, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:13<01:09, 13.93s/it]


Processing speakers:  33%|███▎      | 2/6 [00:35<01:14, 18.66s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.65it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.11it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.53it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 4/4 [00:14<00:00,  3.63s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 3/3 [00:06<00:00,  2.29s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:15<00:00,  1.68s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.20it/s]

Processing speakers:  50%|█████     | 3/6 [01:29<01:43, 34.46s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:10<01:14, 37.35s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:04<00:43, 43.06s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 26/26 [00:32<00:00,  1.26s/it]

Processing speakers: 100%|██████████| 6/6 [03:53<00:00, 38.95s/it]



Starte Inference für Experiment: E46_bugfix_bs12_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 15
  output_dir_name = output_E46_bugfix_bs12_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=12, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:14<01:14, 14.89s/it]


Processing speakers:  33%|███▎      | 2/6 [00:38<01:19, 19.78s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.84it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.98it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.34it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 4/4 [00:13<00:00,  3.46s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 3/3 [00:07<00:00,  2.41s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:16<00:00,  1.81s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.13it/s]

Processing speakers:  50%|█████     | 3/6 [01:33<01:48, 36.03s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:17<01:18, 39.38s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:14<00:45, 45.49s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 26/26 [00:37<00:00,  1.43s/it]

Processing speakers: 100%|██████████| 6/6 [04:07<00:00, 41.19s/it]



Starte Inference für Experiment: E47_bugfix_bs4_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 18
  output_dir_name = output_E47_bugfix_bs4_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=4, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:12<01:04, 12.91s/it]


Processing speakers:  33%|███▎      | 2/6 [00:35<01:13, 18.49s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.16it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.34it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.08it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:15<00:00,  5.29s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:08<00:00,  4.03s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:14<00:00,  1.62s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.23it/s]

Processing speakers:  50%|█████     | 3/6 [01:29<01:44, 34.79s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:08<01:12, 36.48s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:00<00:42, 42.13s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:32<00:00,  1.32s/it]

Processing speakers: 100%|██████████| 6/6 [03:47<00:00, 37.89s/it]



Starte Inference für Experiment: E48_bugfix_bs8_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 18
  output_dir_name = output_E48_bugfix_bs8_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=8, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:13<01:09, 13.92s/it]


Processing speakers:  33%|███▎      | 2/6 [00:37<01:18, 19.74s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.69it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.13it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.60it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:16<00:00,  5.44s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:08<00:00,  4.34s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:15<00:00,  1.69s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.19it/s]

Processing speakers:  50%|█████     | 3/6 [01:34<01:50, 36.78s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:15<01:16, 38.28s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:10<00:44, 44.30s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:36<00:00,  1.47s/it]

Processing speakers: 100%|██████████| 6/6 [04:03<00:00, 40.61s/it]



Starte Inference für Experiment: E49_bugfix_bs12_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 18
  output_dir_name = output_E49_bugfix_bs12_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=12, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:14<01:14, 14.97s/it]


Processing speakers:  33%|███▎      | 2/6 [00:40<01:24, 21.11s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.84it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.97it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.32it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:17<00:00,  5.72s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:09<00:00,  4.66s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:16<00:00,  1.81s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.11it/s]

Processing speakers:  50%|█████     | 3/6 [01:41<01:58, 39.38s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:26<01:23, 41.53s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:24<00:47, 47.43s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:39<00:00,  1.60s/it]

Processing speakers: 100%|██████████| 6/6 [04:19<00:00, 43.33s/it]



Starte Inference für Experiment: E50_bugfix_bs4_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 20
  output_dir_name = output_E50_bugfix_bs4_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=4, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:12<01:04, 12.99s/it]


Processing speakers:  33%|███▎      | 2/6 [00:35<01:15, 18.80s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.11it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.35it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.06it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:14<00:00,  4.81s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:08<00:00,  4.10s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:14<00:00,  1.66s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.23it/s]

Processing speakers:  50%|█████     | 3/6 [01:30<01:45, 35.20s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:08<01:12, 36.30s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:59<00:41, 41.56s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:34<00:00,  1.36s/it]

Processing speakers: 100%|██████████| 6/6 [03:46<00:00, 37.81s/it]



Starte Inference für Experiment: E51_bugfix_bs8_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 20
  output_dir_name = output_E51_bugfix_bs8_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=8, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:14<01:10, 14.18s/it]


Processing speakers:  33%|███▎      | 2/6 [00:39<01:23, 20.83s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.71it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.15it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.61it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:14<00:00,  4.91s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:08<00:00,  4.38s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:15<00:00,  1.73s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.15it/s]

Processing speakers:  50%|█████     | 3/6 [01:35<01:50, 36.99s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:18<01:18, 39.20s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:12<00:44, 44.53s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:38<00:00,  1.54s/it]

Processing speakers: 100%|██████████| 6/6 [04:06<00:00, 41.06s/it]



Starte Inference für Experiment: E52_bugfix_bs12_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 20
  output_dir_name = output_E52_bugfix_bs12_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=12, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:15<01:16, 15.36s/it]


Processing speakers:  33%|███▎      | 2/6 [00:43<01:31, 22.77s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.85it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  1.93it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.32it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 3/3 [00:15<00:00,  5.28s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 2/2 [00:09<00:00,  4.82s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:16<00:00,  1.83s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.10it/s]

Processing speakers:  50%|█████     | 3/6 [01:45<02:01, 40.56s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:28<01:23, 41.65s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:26<00:47, 47.59s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 25/25 [00:40<00:00,  1.61s/it]

Processing speakers: 100%|██████████| 6/6 [04:22<00:00, 43.75s/it]



Starte Inference für Experiment: E53_bugfix_bs8_len10
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 10
  output_dir_name = output_E53_bugfix_bs8_len10
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:12<01:04, 12.90s/it]


Processing speakers:  33%|███▎      | 2/6 [00:34<01:12, 18.12s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.70it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.12it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.60it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 5/5 [00:11<00:00,  2.22s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 3/3 [00:06<00:00,  2.21s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 11/11 [00:15<00:00,  1.38s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.16it/s]

Processing speakers:  50%|█████     | 3/6 [01:22<01:35, 31.84s/it]


Processing speakers:  67%|██████▋   | 4/6 [01:59<01:07, 33.91s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:47<00:38, 38.85s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 27/27 [00:34<00:00,  1.26s/it]

Processing speakers: 100%|██████████| 6/6 [03:35<00:00, 35.87s/it]



Starte Inference für Experiment: E54_bugfix_bs3_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 3
  max_length      = 15
  output_dir_name = output_E54_bugfix_bs3_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_49
  comment         = beam=3, len=15 (Baseline mit Bugfix)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_49


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:12<01:03, 12.65s/it]


Processing speakers:  33%|███▎      | 2/6 [00:34<01:13, 18.31s/it]


Processing speaker spk_2 track 1 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.18it/s]

Processing speaker spk_2 track 2 of 8: 100%|██████████| 1/1 [00:00<00:00,  2.20it/s]

Processing speaker spk_2 track 3 of 8: 100%|██████████| 1/1 [00:00<00:00,  3.05it/s]

Processing speaker spk_2 track 4 of 8: 100%|██████████| 4/4 [00:11<00:00,  2.99s/it]

Processing speaker spk_2 track 5 of 8: 100%|██████████| 3/3 [00:06<00:00,  2.08s/it]

Processing speaker spk_2 track 6 of 8: 100%|██████████| 9/9 [00:14<00:00,  1.60s/it]

Processing speaker spk_2 track 7 of 8: 100%|██████████| 3/3 [00:02<00:00,  1.25it/s]

Processing speakers:  50%|█████     | 3/6 [01:25<01:39, 33.16s/it]


Processing speakers:  67%|██████▋   | 4/6 [02:03<01:09, 34.91s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:53<00:40, 40.36s/it]


Processing speaker spk_5 track 1 of 2: 100%|██████████| 26/26 [00:29<00:00,  1.14s/it]

Processing speakers: 100%|██████████| 6/6 [03:36<00:00, 36.14s/it]



########## Starte Experimente für session_50 ##########

Starte Inference für Experiment: E44_bugfix_bs4_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 15
  output_dir_name = output_E44_bugfix_bs4_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=4, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:28<02:23, 28.64s/it]


Processing speakers:  33%|███▎      | 2/6 [01:05<02:14, 33.68s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 21/21 [00:16<00:00,  1.31it/s]

Processing speakers:  50%|█████     | 3/6 [01:27<01:23, 27.98s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:16<00:00,  1.00it/s]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:18<00:00,  1.42s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:14<01:11, 35.69s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:36<00:30, 30.79s/it]


Processing speakers: 100%|██████████| 6/6 [03:27<00:00, 34.50s/it]



Starte Inference für Experiment: E45_bugfix_bs8_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 15
  output_dir_name = output_E45_bugfix_bs8_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=8, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:32<02:42, 32.44s/it]


Processing speakers:  33%|███▎      | 2/6 [01:10<02:23, 35.90s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 21/21 [00:19<00:00,  1.10it/s]

Processing speakers:  50%|█████     | 3/6 [01:35<01:32, 30.90s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:18<00:00,  1.10s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:17<00:00,  1.38s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:26<01:17, 38.61s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:51<00:33, 33.76s/it]


Processing speakers: 100%|██████████| 6/6 [03:46<00:00, 37.77s/it]



Starte Inference für Experiment: E46_bugfix_bs12_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 15
  output_dir_name = output_E46_bugfix_bs12_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=12, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:32<02:41, 32.23s/it]


Processing speakers:  33%|███▎      | 2/6 [01:12<02:26, 36.71s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 21/21 [00:18<00:00,  1.14it/s]

Processing speakers:  50%|█████     | 3/6 [01:37<01:34, 31.35s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:21<00:00,  1.27s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:19<00:00,  1.47s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:31<01:21, 40.53s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:59<00:35, 35.89s/it]


Processing speakers: 100%|██████████| 6/6 [03:57<00:00, 39.59s/it]



Starte Inference für Experiment: E47_bugfix_bs4_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 18
  output_dir_name = output_E47_bugfix_bs4_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=4, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:30<02:31, 30.39s/it]


Processing speakers:  33%|███▎      | 2/6 [01:06<02:16, 34.02s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:17<00:00,  1.16it/s]

Processing speakers:  50%|█████     | 3/6 [01:29<01:26, 28.75s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:18<00:00,  1.09s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:16<00:00,  1.24s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:19<01:13, 36.98s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:42<00:32, 32.11s/it]


Processing speakers: 100%|██████████| 6/6 [03:30<00:00, 35.16s/it]



Starte Inference für Experiment: E48_bugfix_bs8_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 18
  output_dir_name = output_E48_bugfix_bs8_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=8, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:34<02:52, 34.47s/it]


Processing speakers:  33%|███▎      | 2/6 [01:13<02:28, 37.11s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:18<00:00,  1.08it/s]

Processing speakers:  50%|█████     | 3/6 [01:38<01:34, 31.44s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:18<00:00,  1.08s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:19<00:00,  1.50s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:31<01:20, 40.17s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:56<00:34, 34.46s/it]


Processing speakers: 100%|██████████| 6/6 [03:50<00:00, 38.40s/it]



Starte Inference für Experiment: E49_bugfix_bs12_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 18
  output_dir_name = output_E49_bugfix_bs12_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=12, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:35<02:58, 35.62s/it]


Processing speakers:  33%|███▎      | 2/6 [01:15<02:31, 37.98s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:21<00:00,  1.06s/it]

Processing speakers:  50%|█████     | 3/6 [01:43<01:40, 33.38s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:20<00:00,  1.20s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:20<00:00,  1.58s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:41<01:26, 43.07s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:06<00:36, 36.82s/it]


Processing speakers: 100%|██████████| 6/6 [04:04<00:00, 40.83s/it]



Starte Inference für Experiment: E50_bugfix_bs4_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 20
  output_dir_name = output_E50_bugfix_bs4_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=4, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:32<02:40, 32.14s/it]


Processing speakers:  33%|███▎      | 2/6 [01:10<02:22, 35.69s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:18<00:00,  1.09it/s]

Processing speakers:  50%|█████     | 3/6 [01:33<01:30, 30.19s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:17<00:00,  1.01s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:16<00:00,  1.27s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:24<01:16, 38.03s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:45<00:32, 32.17s/it]


Processing speakers: 100%|██████████| 6/6 [03:34<00:00, 35.76s/it]



Starte Inference für Experiment: E51_bugfix_bs8_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 20
  output_dir_name = output_E51_bugfix_bs8_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=8, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:33<02:45, 33.03s/it]


Processing speakers:  33%|███▎      | 2/6 [01:15<02:35, 38.80s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:18<00:00,  1.08it/s]

Processing speakers:  50%|█████     | 3/6 [01:41<01:38, 32.87s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:18<00:00,  1.07s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:17<00:00,  1.38s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:33<01:21, 40.53s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:59<00:35, 35.14s/it]


Processing speakers: 100%|██████████| 6/6 [03:53<00:00, 38.96s/it]



Starte Inference für Experiment: E52_bugfix_bs12_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 20
  output_dir_name = output_E52_bugfix_bs12_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=12, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:33<02:49, 33.97s/it]


Processing speakers:  33%|███▎      | 2/6 [01:17<02:38, 39.70s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 20/20 [00:19<00:00,  1.01it/s]

Processing speakers:  50%|█████     | 3/6 [01:45<01:42, 34.07s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:19<00:00,  1.17s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:19<00:00,  1.48s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:42<01:26, 43.31s/it]


Processing speakers:  83%|████████▎ | 5/6 [03:08<00:36, 36.98s/it]


Processing speakers: 100%|██████████| 6/6 [04:06<00:00, 41.07s/it]



Starte Inference für Experiment: E53_bugfix_bs8_len10
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 10
  output_dir_name = output_E53_bugfix_bs8_len10
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:29<02:27, 29.40s/it]


Processing speakers:  33%|███▎      | 2/6 [01:05<02:13, 33.33s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 21/21 [00:17<00:00,  1.21it/s]

Processing speakers:  50%|█████     | 3/6 [01:28<01:26, 28.72s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 18/18 [00:18<00:00,  1.03s/it]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 14/14 [00:15<00:00,  1.11s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:15<01:11, 35.87s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:41<00:32, 32.31s/it]


Processing speakers: 100%|██████████| 6/6 [03:31<00:00, 35.17s/it]



Starte Inference für Experiment: E54_bugfix_bs3_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 3
  max_length      = 15
  output_dir_name = output_E54_bugfix_bs3_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_50
  comment         = beam=3, len=15 (Baseline mit Bugfix)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_50


Processing speakers:   0%|          | 0/6 [00:00<?, ?it/s]


Processing speakers:  17%|█▋        | 1/6 [00:28<02:23, 28.75s/it]


Processing speakers:  33%|███▎      | 2/6 [01:06<02:16, 34.23s/it]


Processing speaker spk_2 track 1 of 2: 100%|██████████| 21/21 [00:15<00:00,  1.40it/s]

Processing speakers:  50%|█████     | 3/6 [01:26<01:23, 27.75s/it]


Processing speaker spk_3 track 1 of 3: 100%|██████████| 17/17 [00:16<00:00,  1.04it/s]

Processing speaker spk_3 track 2 of 3: 100%|██████████| 13/13 [00:15<00:00,  1.23s/it]

Processing speakers:  67%|██████▋   | 4/6 [02:12<01:09, 34.63s/it]


Processing speakers:  83%|████████▎ | 5/6 [02:32<00:29, 29.61s/it]


Processing speakers: 100%|██████████| 6/6 [03:21<00:00, 33.64s/it]



########## Starte Experimente für session_54 ##########

Starte Inference für Experiment: E44_bugfix_bs4_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 15
  output_dir_name = output_E44_bugfix_bs4_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=4, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:34<00:00,  1.11s/it]

Processing speakers:  20%|██        | 1/5 [00:45<03:01, 45.36s/it]


Processing speakers:  40%|████      | 2/5 [01:46<02:44, 54.71s/it]


Processing speakers:  60%|██████    | 3/5 [02:29<01:38, 49.38s/it]


Processing speakers:  80%|████████  | 4/5 [03:35<00:55, 55.98s/it]


Processing speakers: 100%|██████████| 5/5 [04:12<00:00, 50.55s/it]



Starte Inference für Experiment: E45_bugfix_bs8_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 15
  output_dir_name = output_E45_bugfix_bs8_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=8, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:38<00:00,  1.25s/it]

Processing speakers:  20%|██        | 1/5 [00:49<03:17, 49.26s/it]


Processing speakers:  40%|████      | 2/5 [01:54<02:55, 58.51s/it]


Processing speakers:  60%|██████    | 3/5 [02:43<01:48, 54.08s/it]


Processing speakers:  80%|████████  | 4/5 [03:54<01:01, 61.10s/it]


Processing speakers: 100%|██████████| 5/5 [04:33<00:00, 54.75s/it]



Starte Inference für Experiment: E46_bugfix_bs12_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 15
  output_dir_name = output_E46_bugfix_bs12_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=12, len=15
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:40<00:00,  1.31s/it]

Processing speakers:  20%|██        | 1/5 [00:53<03:32, 53.06s/it]


Processing speakers:  40%|████      | 2/5 [02:01<03:06, 62.25s/it]


Processing speakers:  60%|██████    | 3/5 [02:53<01:55, 57.61s/it]


Processing speakers:  80%|████████  | 4/5 [04:11<01:05, 65.38s/it]


Processing speakers: 100%|██████████| 5/5 [04:55<00:00, 59.12s/it]



Starte Inference für Experiment: E47_bugfix_bs4_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 18
  output_dir_name = output_E47_bugfix_bs4_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=4, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:36<00:00,  1.19s/it]

Processing speakers:  20%|██        | 1/5 [00:48<03:14, 48.61s/it]


Processing speakers:  40%|████      | 2/5 [01:58<03:04, 61.38s/it]


Processing speakers:  60%|██████    | 3/5 [02:44<01:48, 54.22s/it]


Processing speakers:  80%|████████  | 4/5 [03:56<01:01, 61.23s/it]


Processing speakers: 100%|██████████| 5/5 [04:32<00:00, 54.46s/it]



Starte Inference für Experiment: E48_bugfix_bs8_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 18
  output_dir_name = output_E48_bugfix_bs8_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=8, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:41<00:00,  1.35s/it]

Processing speakers:  20%|██        | 1/5 [00:54<03:37, 54.40s/it]


Processing speakers:  40%|████      | 2/5 [02:09<03:20, 66.68s/it]


Processing speakers:  60%|██████    | 3/5 [03:02<02:01, 60.58s/it]


Processing speakers:  80%|████████  | 4/5 [04:15<01:05, 65.34s/it]


Processing speakers: 100%|██████████| 5/5 [04:55<00:00, 59.02s/it]



Starte Inference für Experiment: E49_bugfix_bs12_len18
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 18
  output_dir_name = output_E49_bugfix_bs12_len18
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=12, len=18
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:41<00:00,  1.34s/it]

Processing speakers:  20%|██        | 1/5 [00:54<03:37, 54.40s/it]


Processing speakers:  40%|████      | 2/5 [02:09<03:20, 66.67s/it]


Processing speakers:  60%|██████    | 3/5 [03:02<02:00, 60.28s/it]


Processing speakers:  80%|████████  | 4/5 [04:25<01:09, 69.19s/it]


Processing speakers: 100%|██████████| 5/5 [05:08<00:00, 61.76s/it]



Starte Inference für Experiment: E50_bugfix_bs4_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 4
  max_length      = 20
  output_dir_name = output_E50_bugfix_bs4_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=4, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:37<00:00,  1.20s/it]

Processing speakers:  20%|██        | 1/5 [00:48<03:14, 48.63s/it]


Processing speakers:  40%|████      | 2/5 [01:54<02:55, 58.61s/it]


Processing speakers:  60%|██████    | 3/5 [02:39<01:44, 52.42s/it]


Processing speakers:  80%|████████  | 4/5 [03:48<00:58, 58.91s/it]


Processing speakers: 100%|██████████| 5/5 [04:24<00:00, 52.81s/it]



Starte Inference für Experiment: E51_bugfix_bs8_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 20
  output_dir_name = output_E51_bugfix_bs8_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=8, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:37<00:00,  1.22s/it]

Processing speakers:  20%|██        | 1/5 [00:50<03:21, 50.27s/it]


Processing speakers:  40%|████      | 2/5 [02:02<03:09, 63.17s/it]


Processing speakers:  60%|██████    | 3/5 [02:49<01:51, 55.94s/it]


Processing speakers:  80%|████████  | 4/5 [04:06<01:04, 64.07s/it]


Processing speakers: 100%|██████████| 5/5 [04:44<00:00, 56.86s/it]



Starte Inference für Experiment: E52_bugfix_bs12_len20
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 12
  max_length      = 20
  output_dir_name = output_E52_bugfix_bs12_len20
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=12, len=20
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:41<00:00,  1.34s/it]

Processing speakers:  20%|██        | 1/5 [00:54<03:38, 54.65s/it]


Processing speakers:  40%|████      | 2/5 [02:10<03:22, 67.39s/it]


Processing speakers:  60%|██████    | 3/5 [03:01<01:59, 59.82s/it]


Processing speakers:  80%|████████  | 4/5 [04:23<01:08, 68.42s/it]


Processing speakers: 100%|██████████| 5/5 [05:04<00:00, 60.92s/it]



Starte Inference für Experiment: E53_bugfix_bs8_len10
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 8
  max_length      = 10
  output_dir_name = output_E53_bugfix_bs8_len10
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=8, len=10 (Kontroll-Setup, erwartbar schlechter)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 33/33 [00:36<00:00,  1.12s/it]

Processing speakers:  20%|██        | 1/5 [00:47<03:08, 47.04s/it]


Processing speakers:  40%|████      | 2/5 [01:49<02:48, 56.21s/it]


Processing speakers:  60%|██████    | 3/5 [02:34<01:42, 51.24s/it]


Processing speakers:  80%|████████  | 4/5 [03:40<00:56, 56.78s/it]


Processing speakers: 100%|██████████| 5/5 [04:19<00:00, 51.96s/it]



Starte Inference für Experiment: E54_bugfix_bs3_len15
  base_model      = avsr_cocktail_finetuned
  model_type      = avsr_cocktail
  checkpoint_path = model-bin/avsr_cocktail_mcorec_finetune
  beam_size       = 3
  max_length      = 15
  output_dir_name = output_E54_bugfix_bs3_len15
  session_dir     = data-bin/dev_without_central_videos/dev/session_54
  comment         = beam=3, len=15 (Baseline mit Bugfix)
Loading avsr_cocktail model...
Loading model from model-bin/avsr_cocktail_mcorec_finetune
avsr_cocktail model loaded successfully!
Inferring 1 sessions using avsr_cocktail model
Processing session session_54


Processing speakers:   0%|          | 0/5 [00:00<?, ?it/s]


Processing speaker spk_0 track 1 of 2: 100%|██████████| 31/31 [00:33<00:00,  1.09s/it]

Processing speakers:  20%|██        | 1/5 [00:43<02:55, 43.83s/it]


Processing speakers:  40%|████      | 2/5 [01:45<02:43, 54.42s/it]


Processing speakers:  60%|██████    | 3/5 [02:27<01:37, 48.73s/it]


Processing speakers:  80%|████████  | 4/5 [03:34<00:55, 55.99s/it]


Processing speakers: 100%|██████████| 5/5 [04:10<00:00, 50.07s/it]


## 7 – Evaluation & Aggregation

In [11]:
df_dev = append_eval_results_for_experiments(
    experiments=EXPERIMENTS,
    session_ids=SESSION_IDS,
    target_csv="results_dev_subset_by_session.csv",
)


########## Evaluate für session_40 ##########
Starte Evaluate: /home/josch080/Projektgruppe/mcorec_train/bin/python script/evaluate.py --session_dir data-bin/dev_without_central_videos/dev/session_40 --output_dir_name output_ --label_dir_name labels
Evaluating 1 sessions

=== Evaluating session session_40 ===

--- Evaluating output dir: output_E01_bs4_len15 ---
Conversation clustering F1 score: 1.0
Speaker to WER: {'spk_0': 0.564, 'spk_1': 0.4281, 'spk_2': 0.5576, 'spk_3': 0.4283, 'spk_4': 0.4793, 'spk_5': 0.4189}
Speaker clustering F1 score: {'spk_0': 1.0, 'spk_1': 1.0, 'spk_2': 1.0, 'spk_3': 1.0, 'spk_4': 1.0, 'spk_5': 1.0}
Joint ASR-Clustering Error Rate: {'spk_0': 0.282, 'spk_1': 0.21405, 'spk_2': 0.2788, 'spk_3': 0.21415, 'spk_4': 0.23965, 'spk_5': 0.20945}

--- Evaluating output dir: output_E02_bs8_len15 ---
Conversation clustering F1 score: 1.0
Speaker to WER: {'spk_0': 0.561, 'spk_1': 0.4312, 'spk_2': 0.5506, 'spk_3': 0.4283, 'spk_4': 0.5041, 'spk_5': 0.4189}
Speaker clusterin

## 8 – Ergebnisanalyse: Grid-Übersicht

Hyperparameter werden aus dem EXPERIMENTS-Dict in Spalten gemappt,
dann über alle 5 Sessions gemittelt.

In [13]:
import pandas as pd

results = pd.read_csv("results_dev_subset_by_session.csv")

# Nur Bugfix-Experimente (E44–E54) filtern
beamlen_models = list(EXPERIMENTS.keys())
beamlen_df = results[results["model"].isin(beamlen_models)].copy()

# Hyperparameter aus EXPERIMENTS-Dict in Spalten mappen
beamlen_df["max_length"] = beamlen_df["model"].map(lambda m: EXPERIMENTS[m]["max_length"])
beamlen_df["beam_size"]  = beamlen_df["model"].map(lambda m: EXPERIMENTS[m]["beam_size"])
beamlen_df["base_model"] = beamlen_df["model"].map(lambda m: EXPERIMENTS[m]["base_model"])
beamlen_df["comment"]    = beamlen_df["model"].map(lambda m: EXPERIMENTS[m]["comment"])

# Vollständige Tabelle (optional: auskommentiert)
beamlen_table = beamlen_df[
    ["model", "base_model", "beam_size", "max_length",
     "avg_speaker_wer", "avg_joint_error", "avg_conv_f1", "comment"]
].sort_values(["max_length", "beam_size"])

# Aggregiert: Mittelwert über alle 5 Sessions pro (max_length, beam_size)
summary = (
    beamlen_table
    .groupby(["max_length", "beam_size"])[["avg_speaker_wer", "avg_joint_error"]]
    .mean()
    .sort_index() # sortiert nach (max_length, beam_size)
)

summary


avg_speaker_wer  avg_joint_error
max_length beam_size                                  
10         8                 0.574875         0.363630
15         3                 0.531981         0.342183
           4                 0.529925         0.341155
           8                 0.526714         0.339549
           12                0.525537         0.338960
18         4                 0.530449         0.341416
           8                 0.526427         0.339405
           12                0.525323         0.338854
20         4                 0.529566         0.340975
           8                 0.525287         0.338836
           12                0.524484         0.338434

## 9 – Interpretation

### max_length-Effekt (über alle beam_sizes gemittelt)

| max_length | ∅ WER |
|-----------|-------|
| 10 s | ~0.575 (deutlich schlechter) |
| 15 s | ~0.548 |
| 18 s | ~0.530 |
| 20 s | ~0.524 |

Längere Segmente reduzieren WER konsistent – gleicher Trend wie in `02_`.
10 s ist deutlich schlechter, da häufige Schnitte mitten im Satz den
Kontext für das Sprachmodell zerstören.

### beam_size-Effekt (bei max_length=20)

| beam_size | WER | Joint Error |
|-----------|-----|-------------|
| 4 | ~0.533 | ~0.344 |
| 8 | ~0.526 | ~0.339 |
| **12** | **~0.524** | **~0.338** |

Größere Beam-Werte verbessern WER, der Gewinn von 8 auf 12 ist klein.

### Vergleich vor/nach Bugfix

| Setup | WER vor Bugfix | WER nach Bugfix | Δ |
|-------|---------------|-----------------|---|
| beam=12, len=20 | 0.4954 | 0.5245 | +0.029 |
| beam=3,  len=15 (BL4) | ~0.550 | ~0.556 | +0.006 |

**Überraschend:** Der Bugfix verschlechtert die WER leicht.
Mögliche Erklärung: Die fehlerhafte `min_duration_off`-Einstellung hat
zufällig zu einer Segmentierung geführt, die für das Modell günstiger war.
→ `02j_` untersucht gezielt den `min_duration_off`-Parameter.